.vscode/settings.json

```diff
@@ -21,6 +21,11 @@
     "files.insertFinalNewline": true,
     "files.trimFinalNewlines": true,
     "files.trimTrailingWhitespace": true,
+    "git.ignoredRepositories": [
+        "benchmarks/google-benchmark",
+        "llvm-project",
+        "boost-math"

```

Comment:
I recommend listing directories in sorted order. I'll push a change.

Reply:
On a related, pre-existing note, `files.exclude` added `boost-math` in non-sorted order. I'll sort that one too, then it'll be clear that these are listing the same things (and `files.exclude` additionally mentions `stl/msbuild`).